controllers/synapse/synapse_controller.go

```diff
@@ -70,8 +70,10 @@ type HomeserverPgsqlDatabase struct {
 func (r *SynapseReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	log := ctrllog.FromContext(ctx)
 
-	// Load the Synapse by name
 	var synapse synapsev1alpha1.Synapse
+	var createdConfigMap corev1.ConfigMap
```

Comment:
Here is another name that made me a bit confused. I have the feeling that without proper comments and explanation I won't understand much what is the goal here. We have a pre-existent ConfigMap, a Copied ConfigMap and also a createdConfigMap. That may be ok if those three versions are needed but I'm not reading it straightforward on the code. I got a bit lost here.

Reply:
Awesome! That's way better.